superset/db_engine_specs/base.py

```diff
@@ -92,6 +92,12 @@
 
 logger = logging.getLogger()
 
+# When connecting to a database it's hard to catch specific exceptions, since we support
+# more than 50 different database drivers. Usually the try/except block will catch the
+# generic `Exception` class, which requires a pylint disablee comment. To make it clear
+# that we know this is a necessary evil we create an alias, and catch it instead.
+GenericDBException = Exception
```

Comment:
why not ->
```python 
class GenericDBException(Exception):
    pass
```

Reply:
nevermind, I understand now reading the code

Reply:
In that case the exceptions from all the DB API drivers would not be derived from `GenericDBException`, so we wouldn't be able to catch them.

The only exception that is a parent to all the exceptions from the multitude of DB API drivers we support is `Exception` itself (I hope!), which is why we have to write:

```python
try:
    # do something with some dynamically loaded DB API driver
except Exception:  # pylint: disable=broad-except
    pass
```

It's inevitable to have a broad except, because the drivers are loaded dynamically. My alias is just a way of saying "I know this is bad, but what can I do?", but also "I'm excepting an exception here that could be anything".